pkg/ovsdb/monitor_test.go

```diff
@@ -152,98 +130,55 @@ func TestTableUpdate(t *testing.T) {
 
 			assert.EqualValuesf(t, ts.expRowUpdate, row, "[%s test] returned wrong row update, expected %#v, got %#v", name, *ts.expRowUpdate, *row)
 		}
-
 	}
-
 }
 
-func TestMonitorResponse(t *testing.T) {
-	/*
-		dataBase := "OVN_Northbound"
-		tableName := "Logical_Router"
-		allColumns := []string{"enabled", "external_ids", "load_balancer", "name", "nat", "options", "policies", "ports", "static_routes"}
-		columns := [][]string{{"enabled", "external_ids", "load_balancer", "name", "nat", "options", "policies", "ports", "static_routes", "_version"},
-			{"name"}, {}}
-		selects := []*libovsdb.MonitorSelect{nil, {Initial: true}, {Initial: false}}
-
-		byteValue, err := common.ReadFile(baseMonitor + "monitor-data.json")
-		assert.Nil(t, err)
-		values := common.BytesToMapInterface(byteValue)
-		expectedResponse, err := common.MapToEtcdResponse(values)
-		var expectedError error
-		dbServer := &DatabaseMock{
-			Response: expectedResponse,
-			Error:    expectedError,
-		}
-		ch := NewHandler(context.Background(), dbServer)
-		cmp := ovsjson.CondMonitorParameters{}
-		cmp.DatabaseName = dataBase
-		cmp.JsonValue = nil
-		mcr := ovsjson.MonitorCondRequest{}
-		for _, sel := range selects {
-			for _, col := range columns {
-				mcr.Columns = col
-				mcr.Select = sel
-				cmp.MonitorCondRequests = map[string][]ovsjson.MonitorCondRequest{tableName: {mcr}}
-
-				monitorResponse, actualError := ch.Monitor(nil, cmp)
-				assert.Nilf(t, actualError, "handler call should not return error: %v", err)
-
-				monitorCondResponse, actualError := ch.MonitorCond(nil, cmp)
-				assert.Nilf(t, actualError, "handler call should not return error: %v", err)
+func TestMonitor(t *testing.T) {
```

Comment:
I rather that you break up this monolith test into a set of smaller tests each checking a specific feature, to eliminate code duplication just use utility functions. Basically you what that is a specific feature of monitor breaks then according to the test name you will be able to know what happened